cmd/oras/internal/option/remote.go

```diff
@@ -87,8 +87,8 @@ func (opts *Remote) ApplyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description
 	}
 }
 
-// ReadPassword tries to read password with optional cmd prompt.
-func (opts *Remote) ReadPassword() (err error) {
+// Parse tries to read password with optional cmd prompt.
+func (opts *Remote) Parse() error {
```

Comment:
The method name `Parse` is reasonable for `Platform` but not for `Remote`.

How about
```go
// Parse bla bla bla
func (opts *Remote) Parse() error {
    return opts.readPassword()
}

// readPassword tries to read password with optional cmd prompt.
func (opts *Remote) readPassword() error {
    // ...
}
```